app/code/community/Qenta/CheckoutPage/Block/Admin/Support/Edit/Form.php

```diff
@@ -40,28 +40,28 @@ protected function _prepareForm()
                 'method'  => 'post'
             )
         );
-        $fieldset = $form->addFieldset('form_form', array('legend' => Mage::helper('wirecard_checkoutpage')->__('Item information')));
+        $fieldset = $form->addFieldset('form_form', array('legend' => Mage::helper('qenta_checkoutpage')->__('Item information')));
 
 
         $fieldset->addField('to', 'select', array(
-            'label'    => Mage::helper('wirecard_checkoutpage')->__('To'),
+            'label'    => Mage::helper('qenta_checkoutpage')->__('To'),
             'class'    => 'required-entry',
             'required' => true,
             'name'     => 'to',
             'options'  => array(
-                'support.at@wirecard.com' => 'Support AT',
-                'support@wirecard.com' => 'Support DE'
+                'support@qenta.com' => 'Support AT',
+                'support@qenta.com' => 'Support DE'
```

Comment:
We don't need a select dropdown for the mail address as there is no real selection. Please turn it into a readonly input

Reply:
Resolved